Diagnostic/rsyslog5/omazurelinuxmds.conf

```diff
@@ -25,4 +25,4 @@ $ActionQueueSaveOnShutdown on
 $ActionQueueFileName lad_mdsd_queue
 $ActionQueueDiscardSeverity 8
 local6.* :omazuremds:;fmtfile
-& ~
+& stop
```

Comment:
Hi @hosungsmsft yes, sorry, somehow that didn't get through properly, yes, the change should only be made for rsyslog7 and later. is there a quick way for you to fix that or you want me to do that?